packages/manager/apps/catalog/e2e/FilterAndSearch.e2e.ts

```diff
@@ -70,46 +60,67 @@ const validateProductCategories = async (
 test('should filter results based on Bare Metal Cloud universe', async ({
   page,
 }) => {
-  await clickButtonByName(page, 'filterButton');
-  await clickCheckboxByName(page, 'checkbox-universe-BareMetalCloud');
-  await clickButtonByName(page, 'applyFilterButton');
+  await clickButtonByText(
+    page,
+    searchTranslation.manager_catalog_search_filter_button,
+  );
```

Comment:
```suggestion
  await page.getByText(searchTranslation.manager_catalog_search_filter_button).click();
```

No need to create a function for this